.update/patches

```diff
@@ -1193,6 +1193,17 @@ Patch_8_16()
 		G_EXEC_HOHALT=1 G_EXEC rm linux-image-visionfive2.deb
 		G_EXEC systemctl unmask serial-getty@hvc0
 	fi
+	
+	#CONFIG_GPU_DRIVER Change: https://github.com/MichaIng/DietPi/issues/6262
+	if (( $G_HW_ARCH == 10 && 
+		$G_DIETPI_INSTALL_STAGE == 2))
+	then	
+	
+		G_WHIP_MSG "DietPi GPU driver installations have been reworked for x86 platforms.\nThis resolves issues with Vulkan, Steam and improves overall driver package installations.\n\nOn the next page you will be asked to install a GPU driver for this device. Once completed, select 'Cancel' from the menu to resume DietPi-Updates.\n\nNB: If you have a custom GPU driver you have manually installed on this system, please select 'Custom' from the next page. DietPi will then ignore changing GPU drivers on this system."
+		/boot/dietpi/dietpi-config 2
```

Comment:
We should only show this menu if any GPU driver or Steam is installed already. Would be confusing for users with VMs or headless/server systems. Just not sure how to obtain this info best. Steam software install state and checking for one of the main packages we checked before in the `dietpi-config` menu: `nvidia-driver` `i965-va-driver` `xserver-xorg-video-amdgpu`?

Reply:
We only need to check for those installed drivers. Then we can offer this prompt.
Steam will apply i386 extras during GPU driver install automatically, if Steam is installed.

Reply:
Jep, also `nvidia-driver-libs:i386` is already installed if Steam was installed as of the Nvidia device check we did before during install:
```sh
[[ -e '/sys/module/nvidia/version' ]] && nvidia=('nvidia-driver-libs:i386')
```

Reply:
https://github.com/MichaIng/DietPi/pull/6273/commits/149017fa04efd2fd4f61c93a2756e08cd3bde615